doc/old_and_new_proposal.md

```diff
@@ -662,6 +676,8 @@ following options.
     * `use_needed`  
       The created VG will match the requirements 1:1, so its size
       will be exactly the sum of all the LVs sizes.
+    * `use_vg_size`  
+      The VG will have exactly the size specified in `lvm_vg_size`.
   * `lvm_vg_size`  
     Specifies the predefined size of the LVM volume group.
```

Comment:
Probably it would be good to clarify that this setting only applies when `lvm_vg_strategy == :use_vg_size`.